### ISSUE TITLE
default buidl returns 404, as it includes "buidlbox" in src path

### ISSUE THREAD

**Issue by dolebas** [uport-project/buidlbox#12]
"Npm run deploy" creates src /build/application/static/ in filesystem but specifies src /buidlbox/static in index.html, see details below:

![image](https://user-images.githubusercontent.com/27515901/39425345-36050b6c-4c7b-11e8-8612-28f353171e97.png)

**Comment by KamesCG** [comment]
@dolebas Yes, that is due to me using `gh-pages` to host the current website.

```
{
  "name": "buidlbox",
  "description": "Build",
  "version": "0.0.1",
  "private": false,
  "license": "MIT",
  "homepage": "/buidlbox",
   ....
```
When running the `build` process the `homepage` parameter should be set to `/` instead of `/buidlbox` to remove this error.

Thank you for reporting this issue :) a better solution will have to be implemented soon.

uPort currently hosts our developer website using `gh-pages`, which causes a domain conflict and which is why the `uport-project` is not being used for the demo website. Working with the team to resolve this issue.

At the moment I'm personal account to run demonstrations.

`https://kamescg.github.io/buidlbox`

but that should change to

`https://uport-project.github.io/buidlbox`

once the DNS settings have been resolved.

**Comment by dolebas** [comment]
Thanks for the thorough reply. After switching the homepage parameter, it gracefully deploys to Netlify as well, out of the box.

If this homepage parameter info is added to the readme, I think most users would be happy with that solution in the meantime!
